bom/pom.xml

```diff
@@ -52,6 +52,7 @@
     <truth.version>0.28</truth.version>
     <vertx.version>3.4.1</vertx.version>
     <jmeter.version>3.2</jmeter.version>
+    <guava.version>23.0</guava.version>
```

Comment:
can you check if this also works with 21.0? It is easier for us to use that version ...